.isort.cfg

```diff
@@ -0,0 +1,7 @@
+[isort]
+line_length=120
+multi_line_output=0
+lines_after_imports=2
+add_imports=from __future__ import unicode_literals
+sections=FUTURE,STDLIB,THIRDPARTY,INDICO,FIRSTPARTY,LOCALFOLDER
```

Comment:
the separate INDICO section ensures that `indico_zodbimport` gets its own section. also like this the same file works fine for plugins where indico is kind-of thirdparty but should still have its own section